nodebuilder/tests/sync_test.go

```diff
@@ -159,6 +189,23 @@ func TestSyncFullWithBridge(t *testing.T) {
 	require.NoError(t, err)
 
 	assert.EqualValues(t, h.Commit.BlockID.Hash, sw.GetCoreBlockHashByHeight(ctx, 30))
+
+	err = full.ShareServ.SharesAvailable(ctx, h.DAH)
+	assert.NoError(t, err)
+
+	for {
```

Comment:
same suggestion here about the use of `Retry` instead of a potential infinite loop